app.js

```diff
@@ -14,16 +13,21 @@ connect('http://127.0.0.1:8088', 'user', 'pass')
   .done();
 
 /**
- * Starts Stasis app 'sla' and initiates SLA application after ARI connection.
+ * Waits for a StasisStart event before going into the main SLA module
  * @param {Object} client - Object that contains information from the ARI 
  *   connection.
  */
 function clientLoaded (client) {
   client.start('sla');
-  sla(client)
-    .then(console.log)
-    .catch(errHandler)
-    .done();
+  client.on('StasisStart', function(event, channel) {
+    if(event.args[0] !== 'dialed') {
```

Comment:
I would consider moving this check to a helper method. That way you can easily modify what determines whether or not a channel was dialed.